homeassistant/components/esphome/ffmpeg_proxy.py

```diff
@@ -250,6 +253,21 @@ async def _write_ffmpeg_data(
             if request.transport and not request.transport.is_closing():
                 await writer.write_eof()
 
+    async def _dump_ffmpeg_stdout(
+        self,
+        proc: asyncio.subprocess.Process,
+    ) -> None:
+        assert proc.stdout is not None
+        assert proc.stderr is not None
+
+        try:
+            while self.hass.is_running and (chunk := await proc.stderr.readline()):
+                _LOGGER.debug(
+                    "ffmpeg[%s] output: %s", proc.pid, chunk.decode().rstrip()
+                )
+        except:  # noqa: E722 - subprocess handling is done in _write_ffmpeg_data
```

Comment:
Not a fan of the complete swallow of the exception. This is already a background task so any exceptions should be properly logged. In other words: this try..except can be removed. Just let it explode if things go sideways and not silently swallow it.

Reply:
sounds reasonable :+1: